user-manuals/orders/bopis/order-details-page/README.md

```diff
@@ -1,42 +1,31 @@
+During peak hours or promotional events, brick-and-mortar retail locations often face increased in-store traffic, heightening the risk that the requested item for pick-up may become unavailable due to in-store purchases. In such instances, store associates may need to reject an order or reject an order partially in case the order item inventory is insufficient at the store. Partial order rejection feature can be enabled using the toggle on the settings page.
 
-To reject an order, access the open order page. Select the order card to navigate to the order details page. Choose reasons for rejection. Click on the 'Report an Issue' button to access a pop-up menu bar for selecting rejection reasons. Confirm the rejection by clicking on the save icon.
+To reject an entire order, click on the order card in the `Open` tab to open the order details page. Click on `Reject Order` to reject the order, select an appropriate reason for rejecting the order in the pop-up window that appears and click on the `save` icon. Another pop-up will appear to confirm order rejection, select `Reject` to reject the order. 
 
-{% hint style="info" %}
-Upon rejecting an item, customers will receive an email with alternate fulfillment options, and the rejected order item will be promptly removed from your dashboard.
-{% endhint %}
+To reject an order partially, click on `Report an Issue` available under the particular item in the order that needs to be rejected, select an appropriate reason for rejecting the item in the pop-up window that appears and click on the `save` icon. Another pop-up will appear to confirm item rejection, select `Reject` to reject the item from the order. 
 
-The reasons for rejection are as followed:
+The available reasons for rejection are:
+* **Not in Stock:** If items are out of stock or exhausted at the store, the store associate can select `Not In Stock` as a reason for rejection. In that case the inventory ATP (Available to Promise) and QOH (Quantity on Hand) are set to 0 for that product for the selected store and all open orders with respective items are auto-rejected from the store.
+* **No Reason:** "If store associates cannot fulfill an order due to high in-store demand and want to maintain the current inventory count, they can choose 'No Reason.' Selecting this option releases reserved inventory, reverting ATP to the original count before the order was received."
 
-**a. Not in Stock**
+{% hint style="info" %} Upon rejecting an item or order, customers will receive an email with [alternate fulfillment options](https://docs.hotwax.co/user-guides/orders/bopis/settings-page#order-edit-permissions) of retailer's choice, and the rejected order or order item will be removed from your dashboard. {% endhint %}
```

Comment:
we should use an internal link here instead of internet link since we are linking within user guides